File: Src/API/WebApplication5/Model/Game.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    public class Game
    {
        public int GameId { get; set; }
        public int teamamount { get; set; }
        public ICollection<Team> teams { get; set; }
        public ICollection<EnabledGameLocaties> enabledLocaties { get; set; }
        public Regio regio { get; set; }
        public int regioId { get; set; }
        public long starttijd { get; set; }
    }


    public class EnabledGameLocaties
    {
        public int GameId { get; set; }
        
        public Game game { get; set; }

        public int LocatieId { get; set; }
        public Locatie locatie { get; set; }
    }
}

File: MobApp/app/src/main/java/be/ap/eaict/geocapture/Model/Game.java
package be.ap.eaict.geocapture.Model;

import java.util.List;

public class Game {

    public int id;
    public  Regio regio;
    public  long starttijd;

    public List<Team> teams;
    public  List<Locatie> enabledLocaties;

    public Game(Regio regio, long starttijd, List<Team> teams, List<Locatie> enabledLocaties)
    {
        this.regio = regio;
        this.starttijd = starttijd;
        this.teams = teams;
        this.enabledLocaties = enabledLocaties;

    }


    public Regio getRegio() {
        return regio;
    }

    public long getStarttijd() {
        return starttijd;
    }

    public List<Locatie> getEnabledLocaties() {
        return enabledLocaties;
    }
}
